packages/excel/src/row.ts

```diff
@@ -89,9 +93,44 @@ export abstract class Row {
 		return subrowOffset + SUBROW_HEADER_SIZE + baseOffset
 	}
 
-	protected unknown(opts?: ColumnSeekOptions) {
-		// NOOP - retrieve the column definition to advance to the next column position
-		this.getColumnDefinition(opts)
+	protected unknown(
+		opts?: ColumnSeekOptions,
+	): {type: ColumnDataType; value: string | number | bigint | boolean | null} {
```

Comment:
personal style stuff, try to avoid `null`, i generally standardise around `undefined` for nullish values

Reply:
I'll remove the null with the change you asked for above 🙂